server/views/account/email-signup.jade

```diff
@@ -8,10 +8,18 @@ block content
             .col-sm-6.col-sm-offset-3
               input(type='hidden', name='_csrf', value=_csrf)
               .form-group
-                  input.input-lg.form-control(type='email', name='email', id='email', placeholder='email', autofocus, required, autocomplete="off")
+                  input.input-lg.form-control(type='email', name='email', id='email', placeholder='Enter your email', autofocus, required, autocomplete="off")
               .form-group
-                  input.input-lg.form-control(type='password', name='password', id='password', placeholder='password', required, pattern=".{8,50}", title="Must be at least 8 characters and no longer than 50 characters.")
+                  input.input-lg.form-control(type='email', name='email', id='email', placeholder='Re-enter your email', autofocus, required, autocomplete="off")
               .form-group
                   button.btn.btn-lg.btn-primary.btn-block(type='submit')
                     span.ion-person-add
                     | Signup
+              br
+              p
+               em We will send a verification link to your email, please verify and you are all set!
```

Comment:
@xRahul @theflametrooper Thanks a lot for the early feedbacks.

Please hold on those thoughts until I take off the WIP tag, because your comments will be lost as I am changing stuff here and there.

The PR update will hide the comments sadly.